HHS/Head-Start-TTADP
# [TTAHUB-3437] Modals should be immediately after the control that spawned them

## Description of change
Move _some_ modals near their triggers. A few remain in the app that aren't inline. I will itemize these in another Jira for a larger refactor.

## How to test
Confirm that you can still unlock reports, change an objective's status from the RTR, and delete an event from the training report landing page.

## Issue(s)

* https://jira.acf.gov/browse/TTAHUB-3437


## Checklists

### Every PR



### Before merge to main


### Production Deploy


### After merge/deploy
